docs/UserGuide.md

```diff
@@ -431,19 +421,23 @@ Edits a specific interview in the list of interviews.
 
 ### Feature: Storage
 
-Save all candidate, position and interview records into a data file locally, on your device itself.
+Save information of all candidates, positions and interviews into a data file locally, on your device itself.
 
-When a candidate, position or interview is added, edited or deleted, the change will be done accordingly in the local save file in real time.
+Modification of any information will be recorded immediately.
 
-The data will be saved in separate files: `/data/candidates.json`, `/data/positions.json`, and `/data/interviews.json`.
+They will be saved in `data` folder in separate files: `/data/candidates.json`, `/data/positions.json`, and `/data/interviews.json`.
 
-By replacing it with another save file with the same name, the data will be loaded accordingly into the application, if the data format is valid.
+Note that `data` will be in the same folder as HR Manager.
 
-If any data is invalid, HR Manager will launch without any data entries.
+If any entry from any of the data files is invalid, HR Manager will launch without any data entries.
 
-The candidate, position and interview information will be saved using the JSON format below.
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
![image](https://user-images.githubusercontent.com/61644924/140009803-da2058c4-b1d9-4fd9-acc4-a74375ad8952.png)

Does this work in Intellij preview, cause in github, it shows this

Reply:
None of the emojis show actually, including the one for features. Is there supposed to be another plugin for that?

Reply:
Sorry, I meant that caution wasn't bolded